src/main/java/com/netflix/imflibrary/st0377/IndexTableSegment.java

```diff
@@ -199,6 +199,10 @@ public String toString()
         }
     }
 
+    public CompoundDataTypes.Rational getIndexEditRate() {
```

Comment:
Elsewhere in the code, _CompoundDataTypes.Rational_ is never returned by public methods. Was this a deliberate design choice? Should we continue with that practice? 
In cases where a fraction is returned, it is usually done as either an instance of _Composition.EditRate_, or as a _List<Long>_. For example- [getSampleRate() in FileDescriptor.java](https://github.com/Netflix/photon/blob/6b3b73a59ad875b10482e2dd6e3a5466828faa2c/src/main/java/com/netflix/imflibrary/st0377/header/FileDescriptor.java#L55)

Reply:
CompoundDataTypes.Rational is in a public interface. After discussion, it does not seem that it was intentionally not used. I'll keep it as is.